src/coreclr/jit/emitxarch.cpp

```diff
@@ -9977,10 +9977,12 @@ void emitter::emitDispIns(
  *  Output nBytes bytes of NOP instructions
  */
 
-static BYTE* emitOutputNOP(BYTE* dstRW, size_t nBytes)
+BYTE* emitter::emitOutputNOP(BYTE* dst, size_t nBytes)
 {
     assert(nBytes <= 15);
 
+    BYTE* dstRW = dst + writeableOffset;
+
 #ifndef TARGET_AMD64
     // TODO-X86-CQ: when VIA C3 CPU's are out of circulation, switch to the
     // more efficient real NOP: 0x0F 0x1F +modR/M
```

Comment:
Are via c3 dead yet?

Reply:
No idea, but it's certainly true that comment was written many years ago.

Reply:
From what I and other from the c# discord can find the C3 has SSE but not SSE2 and SSE is a hard requirement for the current jit 
https://github.com/dotnet/runtime/blob/7890ef1a2366f2f07c8b826b9774c9159cbf2979/src/coreclr/vm/codeman.cpp#L1520-L1523
So in theory if this comment is still accurate about there being a better form of nop it could now be used.